### ISSUE TITLE
The notification bell seems to be missing

### ISSUE THREAD

**Issue by Wild-Dog** [gab-ai-inc/gab-dissenter-extension#125]
I really liked that part.

**Comment by Section25** [comment]
I don't think that anybody at Gab/Dissenter cares about Dissenter/Gab Trends. I've been trying to draw attention to this for months and now the last bit of engagement in the extension, share to Gab, also seems to have stopped working. They are too busy playing with new toys to fix the old ones.

It's a pity because Dissenter was the best thing to happen to the internet in years and shook up the mainstream media when it was launched. I'd take out a lifetime Gab Pro if they fixed it and maintained it.

**Comment by Wild-Dog** [comment]
I was worried no one else even noticed. Thank you for your effort, even if it was fruitless.
